queue/queueservice.js

```diff
@@ -40,6 +40,20 @@ var validateTask = function(task) {
   assert(task.deadline instanceof Date, "Expected task.deadline");
 };
 
+/** Priority to constant for use in queue name (should be a string) */
+const PRIORITY_TO_CONSTANT = {
+  high:   '5',
+  normal: '1'
+};
+_.forIn(PRIORITY_TO_CONSTANT, v => assert(typeof v === 'string'));
+
+/** Priority in order of priority from high to low */
+const PRIORITIES = [
```

Comment:
Well... Then the constant becomes implicit, ie. it becomes the index of the element in the array...

I think it would work, even look nice, the downside is that if we change things here add/remove a priority level we really have to keep using the old constants :)
Or write some code to keep legacy constants around...

If accidentally the constant that is used for a level changes, then all pending tasks might be lost... I would argue that sucks :)
So keeping them explicit makes it harder to make mistakes.